### PR TITLE
rmount: init at 1.0.1

### PR DESCRIPTION
**Pull request by Qubasa**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
-  [X] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/filesystems/rmount/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{ stdenv, nmap, jq, cifs-utils, sshfs, fetchFromGitHub, makeWrapper }:
+
+stdenv.mkDerivation rec {
+
+  pname   = "rmount";
+  version = "1.0.1";
+  name    = "rmount-${version}";
```
Comment (aanderse) [inline pkgs/tools/filesystems/rmount/default.nix]: I should have clarified on the original PR: with `pname` and `version` set, `name` is no longer required, as name is always declared as `name = "${pname}-${version}";`

File: pkgs/tools/filesystems/rmount/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  src = fetchFromGitHub rec {
+    rev = "v${version}";
+    owner="Luis-Hebendanz";
+    repo="rmount";
```
Comment (aanderse) [inline pkgs/tools/filesystems/rmount/default.nix]:
```
owner = "Luis-Hebendanz";
repo = "rmount";
```
nitpick, but a single space surrounding `=` would be nice please.

File: pkgs/tools/filesystems/rmount/default.nix [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    cp ${src}/rmount.man $out/share/man/man1/rmount.1
+    cp ${src}/rmount.bash $out/bin/rmount
+    cp ${src}/config.json $out/share/config.json
+    chmod +x $out/bin/rmount
+
+    wrapProgram $out/bin/rmount --prefix PATH : ${stdenv.lib.makeBinPath [ nmap jq cifs-utils sshfs ]}
+  '';
+
+  meta = {
+      homepage = "https://github.com/Luis-Hebendanz/rmount";
```
Comment (aanderse) [inline pkgs/tools/filesystems/rmount/default.nix]: Quotes aren't actually required here.

File: pkgs/tools/filesystems/rmount/default.nix [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    mkdir -p $out/share/man/man1
+    cp ${src}/rmount.man $out/share/man/man1/rmount.1
+    cp ${src}/rmount.bash $out/bin/rmount
+    cp ${src}/config.json $out/share/config.json
+    chmod +x $out/bin/rmount
+
+    wrapProgram $out/bin/rmount --prefix PATH : ${stdenv.lib.makeBinPath [ nmap jq cifs-utils sshfs ]}
+  '';
+
+  meta = {
```
Comment (aanderse) [inline pkgs/tools/filesystems/rmount/default.nix]: Optionally you can save yourself some typing here and add `with stdenv.lib;` here:
```
  meta = with stdenv.lib; {
    homepage = https://github.com/Luis-Hebendanz/rmount;
    description = "Remote mount utility which parses a json file";
    license = stdenv.lib.licenses.mit;
    maintainers = [ stdenv.lib.maintainers.luis ];
    platforms = stdenv.lib.platforms.linux;
  };
```
Please also keep indentation consistent at 2 spaces.

File: maintainers/maintainer-list.nix [hunk 5]
```diff
@@ -2673,6 +2673,11 @@
     github = "lejonet";
     name = "Daniel Kuehn";
   };
+  luis = {
```
Comment (aanderse) [inline maintainers/maintainer-list.nix]: Looks like this commit is no longer needed as you've already been added to the maintainer list :tada:

**Review by aanderse — CHANGES_REQUESTED** [review]

File: pkgs/tools/filesystems/rmount/default.nix [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+  buildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/share/man/man1
+    cp ${src}/rmount.man $out/share/man/man1/rmount.1
+    cp ${src}/rmount.bash $out/bin/rmount
+    cp ${src}/config.json $out/share/config.json
+    chmod +x $out/bin/rmount
```
Comment (peterhoeg) [inline pkgs/tools/filesystems/rmount/default.nix]: You can use ```install`` instead of the explicit mkdir, cp, chmod steps.

Comment (Luis-Hebendanz) [reply pkgs/tools/filesystems/rmount/default.nix]: Ahh nice to know, thanks!

**Review by aanderse — APPROVED** [review]

**Comment by aanderse** [comment]
@Luis-Hebendanz thanks for your contribution! :tada:

**Pull request merged**